src/test-fbp/test-blob-data.txt

```diff
@@ -0,0 +1 @@
+test validator data
```

Comment:
it is. there was a discussion about a binary mode and ascii mode for file/reader and writer, I'll update the test when this is released.